1-exercises/F-for-of-loop/exercise.js

```diff
@@ -10,7 +10,13 @@ let tubeStations = [
     "Oxford Street",
     "Tottenham Court Road"
 ];
+for(let i=0; i<tubeStations.length;i++)
+console.log(tubeStations[i]);
 
 
 // TODO Use a for-of loop to capitalise and output each letter in the string seperately.
 let str = "codeyourfuture";
+let output = "";
+for(let i = 0;i<str.length;i++)
+output+=str[i].toUpperCase();
```

Comment:
You are missing curly brackets { } around line 21 to contain the code you want to be executed within the for loop. I'm guessing it still works because .toUpperCase is a string method, so we technically don't need a loop, we could just use str.toUpperCase() and it will still give us the result we want, but the assignment calls for a loop.

Reply:
You are correct Michelle, by using only the toUpperCase function, we can get the result that we want, but above the solution, they asked to use a loop to capitalize **each letter.** 

Sometimes it is not necessary to use brackets for a single-line statement, but for more than one-line statements it is mandatory, otherwise, the rest of the line of code won't be executed as a part of that particular statement, and my mistake is using for loop instead of for-of-loop, and I need to fix it Thanks for making me aware of it.

Reply:
True, I forgot that part. And that's interesting. I've never seen it written like that, not even in documentation. Are you sure that's not Python you're thinking of instead of JavaScript? You probably don't want to make a habit of doing that, as you said, it could cause you issues later on down the line. 

Either way, happy I could help 👍